bears/general/LineBreakBear.py

```diff
@@ -0,0 +1,133 @@
+from queue import Queue
+
+from coalib.results.AbsolutePosition import AbsolutePosition
+from coalib.results.Diff import Diff
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.SourceRange import SourceRange
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+
+from bears.general.AnnotationBear import AnnotationBear
+from bears.general.RangeHelpers import get_specified_block_range
```

Comment:
um, this is the first commit of your PR, but there is no `/bears/general/RangeHelpers.py` on master. It shows as edited file of this PR, so I guess you create that later? Thats bad, you don't want commits to depend on stuff that happens later in the commit history.